source/fundamentals/atlas-search.txt

```diff
@@ -335,6 +335,28 @@ The search returns the following document:
 To learn more about the ``geoWithin`` operator, see the :atlas:`geoWithin </atlas-search/geoWithin>` 
 Atlas guide.
 
+In
+~~
+
+Use the ``In()`` method to search for documents that have field values matching a list
+of specified values.
+
+The following example searches the ``guitars`` collection for documents that have a
+``make`` field value of either ``"Fender"`` or ``"Gibson"``.
+
+.. literalinclude:: /includes/fundamentals/code-examples/atlas-search/AtlasSearchExamples.cs
+   :start-after: // start-in-search
+   :end-before: // end-in-search
```

Comment:
```suggestion
   :start-after: start-in-search
   :end-before: end-in-search
```

Reply:
A symptom of relying too much on Copilot autocomplete.